docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -242,3 +242,58 @@ As you can see, we get different groups of hosts.
 
 In case a filter parameter is not defined, the plugin supposes all values possible are wanted.
 This means that for each tag that exists on your Scaleway compute nodes, a group based on each tag will be created.
+
+Scaleway S3 object storage
+==========================
+
+Object Storage allows you to store any kind of objects (documents, images, videos, etc.).
+As the Scaleway API is S3 compatible, Ansible supports it natively through the modules: :ref:`s3_bucket_module_module`, :ref:`aws_s3_module`.
+
+You can find many examples in `./test/legacy/roles/scaleway_s3`
```

Comment:
Needs double backticks here.

In reStructuredText: single backticks with `:ref:` make a link; double backticks on their own make a `code quotation`.